skyportal/api_description.md

```diff
@@ -34,6 +34,23 @@ if response.status_code in (200, 400):
 curl -s -H 'Authorization: token ea70a5f0-b321-43c6-96a1-b2de225e0339' http://localhost:5000/api/sysinfo
 ```
 
+#### Using query parameters
+
+```python
+import requests
+
+token = 'ea70a5f0-b321-43c6-96a1-b2de225e0339'
+
+response = requests.get(
+    'http://localhost:5000/api/candidates?numPerPage=100&pageNumber=1',
```

Comment:
Better to use `params` keyword?

Reply:
That's `requests`-specific, but I just want to illustrate how to include them in the URL generally, as this extends to requests generated via other means